x-pack/plugin/apm-integration/src/main/java/org/elasticsearch/xpack/apm/APMTracer.java

```diff
@@ -102,57 +163,58 @@ protected void doStart() {
                 .build()
         );
 
-        openTelemetry = OpenTelemetrySdk.builder()
+        var openTelemetry = OpenTelemetrySdk.builder()
             .setTracerProvider(provider)
             .setPropagators(ContextPropagators.create(W3CTraceContextPropagator.getInstance()))
             .build();
-        tracer = openTelemetry.getTracer("elasticsearch", Version.CURRENT.toString());
+        var tracer = openTelemetry.getTracer("elasticsearch", Version.CURRENT.toString());
+
+        this.services = new APMServices(provider, tracer, openTelemetry);
```

Comment:
```suggestion
        assert this.services == false; // ClusterApplierService stops before we do
        this.services = new APMServices(provider, tracer, openTelemetry);
```

Reply:
Do you mean `assert this.services == null;` or uninitialized before we start?

Reply:
bah yeah I meant `null` sorry